src/olympia/editors/helpers.py

```diff
@@ -557,6 +557,7 @@ def get_actions(self, request, addon):
             acl.action_allowed(request, 'ReviewerAdminTools', 'View'))
         reviewable_because_submission_time = (
             not is_limited_reviewer(request) or
+            not addon.latest_version or
             (datetime.datetime.now() - addon.latest_version.nomination >=
```

Comment:
what about when `latest_version is not None` but `latest_version.nomination `isn't set?